library_generation/generate_library_integration_test.sh

```diff
@@ -105,24 +111,23 @@ echo "Generating library from $proto_path, to $destination_path..."
 --transport "$transport" \
 --rest_numeric_enums "$rest_numeric_enums" \
 --include_samples "$include_samples" \
+--enable_postprocessing "true" \
+--repo_metadata_json_path $repo_metadata_json_path \
+--owlbot_sha '3a95f1b9b1102865ca551b76be51d2bdb850900c4db2f6d79269e7af81ac8f84' \
```

Comment:
We should get owlbot sha from .github/.OwlBot.lock.yaml in google-cloud-java

Reply:
I added inference logic in the post-processing section